openlibrary/plugins/openlibrary/js/scroll.js

```diff
@@ -0,0 +1,17 @@
+$(window).scroll(function() {
+    var height = $(window).scrollTop();
+    if (height > 200) {
+        $('#back2Top').fadeIn();
```

Comment:
Please document why you are fading in and out here.
I'd also like to know in a comment why you chose 200px ? That's quite small on desktop.

Reply:
Yes I realized this now that it is small , changed to 1200. Now I think it is good!

Reply:
Maybe this value should be related to window.innerHeight ?

i.e. window.innerHeight * 2 would be one scroll.

Reply:
Yes this value is related to window.innerHeight. 
So, should I make this height> (window.innerHeight*2)?

Reply:
It's working fine by making this change

Reply:
I think 2 makes sense and makes it easier for a developer to understand where the value comes from. Thanks for the change! :)